source/include/signalflow/node/node-monitor.h

```diff
@@ -1,7 +1,7 @@
 #include "signalflow/node/node.h"
 
 #include <thread>
-#include <unistd.h>
+//#include <unistd.h>
```

Comment:
Feel free just to delete any all of the old lines of code like this rather than commenting - we can always find old code in git history if need be.

Reply:
👍